.github/workflows/test-dbt-installation-main.yml

```diff
@@ -10,17 +10,33 @@ name: dbt Installation Tests
 
 permissions:
   contents: read # required for slack-post-notification workflow
+  packages: read # required to fetch tags for a container
 
 on:
   workflow_dispatch:
+    inputs:
+      test-installation-method:
+        description: "Choice installation method to test"
```

Comment:
```suggestion
        description: "Choose installation method(s) to test"
```

Reply:
Fixed in https://github.com/dbt-labs/actions/pull/67/commits/a0f4fd419532a523dc1e9a91411de1f9ccd8e44f